mq/eventtask.ini

```diff
@@ -0,0 +1,25 @@
+[uwsgi]
+chdir = /opt/mozdef/envs/mozdef/mq/
+uid = mozdef
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+mule = esworker_eventtask.py
+pyargv = -c /opt/mozdef/envs/mozdef/mq/esworker_eventtask.conf
+log-syslog = syslog-worker
```

Comment:
Should 'syslog-worker' be changed to reflect the new 'eventtask' naming scheme?

Reply:
oh! thanks for catching this! I had it changed, but guess I reverted when I backed out a commit.